src/components/ListedSpecies/BehaviorsSection.jsx

```diff
@@ -1,10 +1,12 @@
-import { FormControl, FormLabel, Grid, GridItem } from '@chakra-ui/react';
+import { FormControl, FormLabel, Grid, GridItem, Text } from '@chakra-ui/react';
+import { Select } from 'chakra-react-select';
+import { useFormContext } from 'react-hook-form';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useFormContext' is defined but never used.